README.md

```diff
@@ -39,9 +39,9 @@ To run a local server that displays the generated site, run:
 
 ```bash
 # Rebuild the site whenever relevant files change:
-bundle exec guard
-# Start the local development server:
-bundle exec nanoc view
+$ make guard
+# Start the local development server in a separate shell:
+$ make serve
```

Comment:
I'm not a big fan of putting `$` in front of prompts in Markdown code blocks because it makes it impossible to copy-and-paste the whole block, which is annoying. I think it's a great idea though in environments where the `$` can be displayed in a non-copyable margin.